### ISSUE TITLE
Random sequence check failures

### ISSUE THREAD

**Issue by sorokya** [sorokya/reoserv#1]
Sometimes the server will disconnect the official client for an invalid sequence. Seems like a server bug.

Examples:
* `> player 1 connection closed: sending invalid sequence: Got 58196, expected 1748.`
* `> player 1 connection closed: sending invalid sequence: Got 20777, expected 1373.`

```
 DEBUG reoserv::handle_player                > Recv: [1, 4, 6, 138, 97, 100, 109, 105, 110, 255, 116, 101, 115, 116, 255]
 DEBUG reoserv::handlers::login::request     > Recv: Request { name: "admin", password: "test" }
 INFO  reoserv::handle_player                > player 1 connection closed: sending invalid sequence: Got 34666, expected 1402.
```

```
 DEBUG eo::net::packet_processing            > Decoding packet:
Encoded: [112, 235, 129, 88, 125]
Decode multiple: 7
 DEBUG reoserv::handle_player                > Recv: [240, 1, 253, 216, 107]
 INFO  reoserv::handle_player                > player 1 connection closed: sending invalid sequence: Got 54647, expected 1727.
```

**Comment by sorokya** [comment]
I'm willing to bet this was an issue with the packet encoding swap multiples being backwards. Should have been fixed with fa5d42585fd9f35d549bdaa96fc102b515f56219

**Comment by sorokya** [comment]
Did some testing and confirmed issue is resolved. Made sequence enforcement the default again
